packages/grid/x-data-grid/src/hooks/features/rows/useGridParamsApi.ts

```diff
@@ -58,9 +53,7 @@ export function useGridParamsApi(apiRef: React.MutableRefObject<GridApiCommunity
   const getCellValueWithDeprecationWarning = React.useCallback<GridParamsApi['getCellValue']>(
     (...args) => {
       if (process.env.NODE_ENV !== 'production') {
-        if (!warnedOnceGetValue) {
-          warnGetValue();
-        }
+        getCellValueWarning();
       }
```

Comment:
Why not using the `wrapWithWarningOnCall`?

Reply:
Right now `apiRef.current.getCellValue` reference never changes because it's deps do not change (there is only `apiRef`).
But if in the future we add a new dep to this `useCallback`, the reference might change, creating nasty bugs.